From 36eb03ec029b05a3f68b38af91b318f548b496fb Mon Sep 17 00:00:00 2001
From: William Witt <william@witt-family.net>
Date: Sat, 3 Sep 2011 12:16:19 -0500
Subject: [PATCH 15/18] sandisk_sansa-fuze-plus: Add MSC mode Device ID

Sansa Fuze+ uses a different Device ID for forced MSC mode than
MTP/Auto.

Signed-off-by: Martin Pitt <martin.pitt@ubuntu.com>
---
 media-players/sandisk_sansa-fuze-plus.mpi |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/media-players/sandisk_sansa-fuze-plus.mpi b/media-players/sandisk_sansa-fuze-plus.mpi
index 2d137aa..9d3d086 100644
--- a/media-players/sandisk_sansa-fuze-plus.mpi
+++ b/media-players/sandisk_sansa-fuze-plus.mpi
@@ -2,7 +2,7 @@
 Product=Sansa Fuze+
 Vendor=SanDisk
 AccessProtocol=storage
-DeviceMatch=usb:0781:74e0;
+DeviceMatch=usb:0781:74e0;0781:74e1;
 
 [Media]
 InputFormats=audio/x-wav
-- 
1.7.6

